code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -853,16 +853,17 @@
 	description = "A strong anesthetic and sedative."
 	reagent_state = LIQUID
 	color = "#96DEDE"
+	metabolization_rate = 0.1
 
 /datum/reagent/medicine/ether/on_mob_life(mob/living/M)
 	M.AdjustJitter(-25)
 	switch(current_cycle)
-		if(1 to 15)
+		if(1 to 30)
 			if(prob(7))
 				M.emote("yawn")
-		if(16 to 35)
+		if(31 to 40)
```

Comment:
This phase is now only 9 cycles long, as opposed to the previous 19 cycles it lasted. Sure, they yawn longer, but they drop much faster afterwords.